Documentation/components/drivers/special/index.rst

```diff
@@ -22,9 +22,11 @@ following section.
 .. toctree::
   :caption: Supported Drivers
 
+  syslog.rst
   spi.rst
   i2c.rst
   ethernet.rst
+  socketcan.rst
```

Comment:
@btashton and @v01d do you know why syslog.rst, socketcan.rst can't show on the website?
https://nuttx.apache.org/docs/latest/components/drivers/special/index.html
even the source show that they are there:
https://nuttx.apache.org/docs/latest/_sources/components/drivers/special/index.rst.txt
nsh has the similar issue, but note.rst don't, very strange.

Reply:
https://nuttx.apache.org/docs/latest/components/drivers/special/socketcan.html

I see it here?

Reply:
My sceenshot at https://nuttx.apache.org/docs/latest/components/drivers/special/index.html:
![image](https://user-images.githubusercontent.com/18066964/96417503-f516ce80-1223-11eb-807d-4faa31618bbb.png)
My sceenshot at https://nuttx.apache.org/docs/latest/components/drivers/special/socketcan.html:
![image](https://user-images.githubusercontent.com/18066964/96417636-2394a980-1224-11eb-911f-0cd453276734.png)
Very strange.

Reply:
@xiaoxiang781216 try to do "CTRL+SHIT+R", this forces the browser to reload and not use cache. I've seen this happen some times on Firefox at least. I wonder if we need some extra tag on the HTMLs to make the server know about the changed HTML

Reply:
Yes, reload can work! Thanks, @v01d.